### PR TITLE
Update linters pypi links to latest version

### PR DESCRIPTION
**Pull request by alonme**
I saw that some of the links to the linters PYPI page are not pointing to the latest version - updated those.

BTW - any reason to link to a specific version and not to the latest (https://pypi.org/project/ruff/ instead of https://pypi.org/project/ruff/0.0.228/) ?

### CODE REVIEW

**Comment by charliermarsh** [comment]
Yeah, the versions actually were intentional, since they represent the version at which the plugin was ported over (in the event that they continue to evolve). But, maybe that's just confusing, and not very helpful.

If we do want to change this, though, it needs to be done in Rust files themselves -- so, e.g., we'd need to change this:

```rust
//! Rules from [flake8-tidy-imports](https://pypi.org/project/flake8-tidy-imports/4.8.0/).
```

Then run `cargo dev generate-all` to regenerate the README :)

**Comment by alonme** [comment]
Ok cool,
I found it confusing - but maybe it just needs to say that explicitly,
So whatever you think is best

**Comment by charliermarsh** [comment]
I think it's fine to change it, if you don't mind modifying the references in those source files.

**Comment by alonme** [comment]
by change do you mean update to latest version or delete the version?

**Comment by charliermarsh** [comment]
Ah sorry, I mean remove the version, and point to latest.

**Comment by not-my-profile** [comment]
I think it makes sense to remove the versions since if anything we'd have to track the upstream version for each rule individually ... it doesn't make sense to track it for a whole linter.

**Review by not-my-profile — CHANGES_REQUESTED** [review]
The links should include the trailing `/` because otherwise there's an unnecessary redirect when following these links.

You'll also want to run `cargo dev regenerate-all` to update the links in the README.

**Review by sbrugman — COMMENTED** [review]
Please also update the `scripts/add_plugin.py` example:
https://github.com/charliermarsh/ruff/blob/main/scripts/add_plugin.py#L8

File: src/rules/pygrep_hooks/mod.rs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-//! Rules from [pygrep-hooks](https://github.com/pre-commit/pygrep-hooks).
+//! Rules from [pygrep-hooks](https://github.com/pre-commit/pygrep-hooks/).
```
Comment (not-my-profile) [inline src/rules/pygrep_hooks/mod.rs:1]: nit: but the trailing slash isn't necessary for GitHub

File: src/rules/flake8_import_conventions/mod.rs [hunk 2]
```diff
@@ -1,4 +1,4 @@
-//! Rules from [flake8-import-conventions](https://github.com/joaopalmeiro/flake8-import-conventions).
+//! Rules from [flake8-import-conventions](https://github.com/joaopalmeiro/flake8-import-conventions/).
```
Comment (not-my-profile) [inline src/rules/flake8_import_conventions/mod.rs:1]: same here^^

File: scripts/add_plugin.py [hunk 3]
```diff
@@ -5,7 +5,7 @@
 
     python scripts/add_plugin.py \
         flake8-pie \
-        --url https://pypi.org/project/flake8-pie/0.16.0/
+        --url https://pypi.org/project/flake8-pie/
```
Comment (sbrugman) [inline scripts/add_plugin.py:8]: There is one more reference on line 107

**Comment by charliermarsh** [comment]
Thanks!

**Pull request merged**